.rubocop.yml

```diff
@@ -29,21 +29,11 @@ Layout/LineLength:
     - db/**/*
 
 Layout/SpaceBeforeFirstArg:
-  Exclude:
-    - app/views/api/**/**/*
+  AllowForAlignment: true
```

Comment:
I don't think this is for comments right? I think it's because in views we do 
```ruby
json.id         user.id
json.email      user.email
json.name       user.full_name
json.username   user.username
```

which I don't like but it is what it is 🤷

Reply:
This is indeed only for arguments, not comments. `Layout::CommentIndentation` would be the one that would modify comments